src/test/resources/case_claim_example/suite.xml

```diff
@@ -102,6 +107,18 @@
             </text>
           </display>
         </prompt>
+        <prompt key="dob" required="false()">
```

Comment:
Should having both the attribute and the child element be allowed? Seems like it could be confusing.